### ISSUE TITLE
[TW-456] Exporting output is not always valid

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#491]
_Ben Boeckel on 2012-09-13T20:05:07Z says:_ 

 When exporting to JSON, the line which mentions that a different configuration file is being used is not valid JSON. Arguably, it should go to stderr anyways. This breaks taskwarrior-web at a minimum.

In addition, the "No matches." output should conform to the format being exported (it should probably just not be there).

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-09-13T20:05:07Z
Modified: 2014-02-09T01:54:31Z
```

**Comment by taskwarrior** [comment]
_David Patrick on 2012-09-14T05:40:02Z says:_ 

 just wondering.. is export output run through json validator, as part of flod?

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-09-14T11:42:18Z says:_ 

 Ben, try this:

<pre>
task rc.verbose=nothing rc:<other_file> export
</pre>

David, no there is no validation in the test suite.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-09-16T18:44:48Z says:_ 

 Already fixed in 2.2.0.
